serving/docker/docker-compose.yml

```
@@ -0,0 +1,5 @@
+version: '3'
+services:
+  cpu:
+    build: .
+    image: deepjavalibrary/djl-serving:minimal-nightly
```

We should make the tag configuration
```suggestion
    image: deepjavalibrary/djl-serving:${TAG}
```